modules/admin-ui-picard/app/src/components/users/partials/wizard/EditUserGeneralTab.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {useTranslation} from "react-i18next";
+import Notifications from "../../../shared/Notifications";
+import cn from "classnames";
+import {Field} from "formik";
+
+/**
+ * This component renders the general user information tab in the users details modal.
+ */
+const EditUserGeneralTab = ({ formik }) => {
+    const { t } = useTranslation();
+
+    // style used in user details modal
+    const editStyle = {
+        color: '#666666',
+        fontSize: '14px'
+    }
+
+    // style for input fields when disabled
+    const disabledStyle = {
+        backgroundColor: '#eeeff0'
+    }
+
+    return (
+        <div className="modal-content">
+            <div className="modal-body">
+                <div className="form-container">
+                    <Notifications />
```

Comment:
Sould display a notification, if the user can't be edited `(!formik.values.manageable)` should probably be set in thunk after fetching the user data